morf-integration-test/src/test/java/org/alfasoftware/morf/integration/TestSqlStatements.java

```diff
@@ -2488,10 +2488,10 @@ private void preparedStatementRecord(SqlDialect sqlDialect, NamedParameterPrepar
     sqlDialect.prepareStatementParameters(
       preparedStatement,
       ImmutableList.of(
-        parameter("column1").type(INTEGER),
-        parameter("column2").type(DECIMAL),
-        parameter("column3").type(STRING),
-        parameter("parameterValue").type(STRING)
+        parameter("column1").type(DataType.INTEGER),

```

Comment:
Why are these static imports being removed? I think the DSL usage reads better without the enum name.

Reply:
Most of the other code used DataType.INTEGER instead of static INTEGER,
which also matched my personal preference (apparently opposite to yours :)),
so I unified the usage in this direction.

I seem to prefer static imports for methods, but not constants. Funny..